<?xml version="1.0"?>

<launch>

<node name="osm_particle_filter" pkg="osm-localization" type="pf_node">
    <rosparam command="load" file="$(find osm-localization)/config/config.yaml" />
</node>
</launch>